docs/operate/relayer/monitoring-alerting.mdx

```diff
@@ -0,0 +1,35 @@
+# Monitoring & Alerting
+
+The relayer exposes metrics on the port number specified by the argument `--metrics`. Port `9090` is the default, though any valid port can be chosen. 
+
+We recommend using Prometheus to scrape these metrics and Grafana to visualize them, using [this dashboard JSON template](https://github.com/hyperlane-xyz/hyperlane-monorepo/blob/2f7f714c5b698fedcead9825f52da6ef95f96be1/tools/grafana/easy-relayer-dashboard-template.json), as seen below. The benefit of this dashboard is that it displays multiple validators at the same time. The screenshot shows metrics grouped by chain or by kubernetes pod.
```

Comment:
noting that we'll need to update this permalink if we ever make tweaks - I take it you feel this is preferable to this link 404ing if the path ever changes?

Reply:
I do think this is preferable. If the source code changes, we'd likely need to make changes to the docs too